lib/ui/painting/display_list_deferred_image_gpu.cc

```diff
@@ -64,9 +64,10 @@ SkISize DlDeferredImageGPU::dimensions() const {
 
 // |DlImage|
 size_t DlDeferredImageGPU::GetApproximateByteSize() const {
```

Comment:
I think we should just do this in `image.cc` (`CanvasImage::GetAllocationSize` instead of here. 

There's a feature request somewhere to expose the size of an image through to dart:ui. This method should be left as-is to support that. Alternatively, we should just delete these methods from `DlImage`.

There's already a hacky work around in CanvasImage::GetAllocationSize to try to avoid tripping up the VM when you allocate a really large image. I'm becoming more convinced that we should just stop telling the VM about image sizes.

@rmacnak-google or @jason-simmons might have some better ideas.